docs/components/checkboxes.md

```diff
@@ -21,11 +21,16 @@ Use checkboxes to:
 
 ## Contents
 
-- [Contents](#contents)
-- [Using checkboxes](#using-checkboxes)
-- [Checkboxes](#checkboxes)
-- [Checkbox key properties](#checkbox-key-properties)
-- [Theming](#theming)
+- [Selection controls: checkboxes](#selection-controls-checkboxes)
```

Comment:
Should remove the first and second links here too, no?

Reply:
Done